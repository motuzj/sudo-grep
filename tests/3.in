multiple words search
"elit sed"
