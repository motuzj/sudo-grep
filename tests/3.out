consectetur adipiscing elit sed,
Quis auctor elit sed vulputate mi sit amet.
